service/service.go

```diff
@@ -83,19 +84,30 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
-	g8sClient, err := versioned.NewForConfig(restConfig)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
+	var k8sClient k8sclient.Interface
+	{
+		c := k8sclient.ClientsConfig{
+			AddToScheme: func(s *runtime.Scheme) error {
```

Comment:
Just noticed something: https://github.com/giantswarm/apiextensions/blob/master/pkg/clientset/versioned/fake/register.go#L37-L59

So this could be:
```suggestion
			AddToScheme: runtime.SchemeBuilder{
				corev1alpha1.AddToScheme,
				applicationv1alpha1.AddToScheme,
			},
```

Reply:
Sick. How about adding convenience wrapper/methods in k8sclient so we don't have to remember the import?

Reply:
This is  nice and I think maybe the interface can be improved by passing the scheme builder as a parameter.

WIP PR to get feedback is here. https://github.com/giantswarm/k8sclient/pull/23